<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>EvolutionarySRep</class>
  <widget class="qMRMLWidget">
    <layout class="QVBoxLayout">
      <item>
        <widget class="ctkCollapsibleButton">
          <property name="text">
            <string>General I/O</string>
          </property>
          <layout class="QFormLayout">
            <item row="0" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Input Model</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="qMRMLNodeComboBox" name="inputModel">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="nodeTypes">
                 <stringlist notr="true">
                  <string>vtkMRMLModelNode</string>
                 </stringlist>
                </property>
               </widget>
            </item>
            
            <item row="2" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Output Directory</string>
                </property>
              </widget>
            </item>
            <item row="2" column="1">
              <widget class="ctkDirectoryButton" name="outputPath">
                <property name="toolTip">
                  <string>Output Directory</string>
                </property>
              </widget>
            </item>

          </layout>
        </widget>
      </item>

      <item>
        <widget class="QPushButton" name="ApplyButton">
          <property name="text">
            <string>Apply</string>
          </property>
        </widget>
      </item>

      <item>
        <widget class="QProgressBar" name="fittingProgressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>

      <item row="100" column="0">
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>ctkCollapsibleButton</class>
      <extends>QWidget</extends>
      <header>ctkCollapsibleButton.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>qMRMLWidget</class>
      <extends>QWidget</extends>
      <header>qMRMLWidget.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkSliderWidget</class>
      <extends>QWidget</extends>
      <header>ctkSliderWidget.h</header>
      <container>0</container>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>
